Tools/scripts/deepfreeze.py

```diff
@@ -1,7 +1,9 @@
 """Deep freeze
 
-The script is executed by _bootstrap_python interpreter. Shared library
-extension modules are not available.
+The script may be executed by _bootstrap_python interpreter.
```

Comment:
LGTM, another thing is that even though this script is executed by `_bootstrap_python` with the same magic number, still it uses umarshal rather than directly using `compile` even though it supports both but I suppose that it would be too much info to add.

Reply:
I guess that code could even be removed?

Reply:
That code is useful as sometimes I used it to debug marshal output so better to leave as is.

Reply:
Okay, that's fine.